src/vma/sock/sock-redirect.cpp

```diff
@@ -525,10 +525,8 @@ int vma_add_conf_rule(char *config_line)
 
 	int ret = __vma_parse_config_line(config_line);
 
-#if (VMA_OPTIMIZE_LOG <= 5)
 	if (*g_p_vlogger_level >= VLOG_DEBUG)
 		__vma_print_conf_file(__instance_list);
-#endif
 
```

Comment:
1. g_p_vlogger_level should not be higher than VMA_MAX_DEFINED_LOG_LEVEL (so the C condition is enough)
2. The command __vma_pirnt_conf_file() is not on fast path. So no need to wrap with #if
3. The original implementation seemed wrong. Say VMA_MAX_DEFINED_LOG_LEVEL==7 then you should have printed, but the condition 
#if (VMA_OPTIMIZE_LOG <= 5) will exclude the printing.